server/controllers/notifications.controller.js

```diff
@@ -1,36 +1,23 @@
-import Sequelize from 'sequelize';
-import db from '../../config/sequelize';
-import pushNotificationHelper from '../helpers/pushNotificationHelper';
+/* eslint-disable import/prefer-default-export */
+import { User } from '../models/user.model';
+import { sendPushNotification } from '../helpers/pushNotificationHelper';
 
-const Device = db.Device;
-const User = db.User;
-const Op = Sequelize.Op;
-
-
-function sendPushNotifications(req, res, next) { // eslint-disable-line no-unused-vars
+export async function sendPushNotifications(req, res) {
     const { pushData } = req.body;
 
     const usernames = pushData.map(data => data.username);
 
-    // Find all users specified above, left join on device table
-    User.findAll({
-        include: [Device],
-        where: {
-            username: {
-                [Op.in]: usernames,
-            },
+    const users = await User.find({
+        username: {
+            $in: usernames,
         },
-    }).then((users) => {
-        if (!users) return;
-        users.forEach((user) => {
-            if (!user) return;
-            const userData = pushData.find(_userData => _userData.username === user.username);
-            pushNotificationHelper.sendPushNotification(user, userData);
-        });
-        res.send({ success: true });
     });
-}
+    // TODO how are we doing errors?
+    if (!users) throw new Error('USERS_NOT_FOUND');
```

Comment:
@mountHouli I imagine this will need to be changed based on our new Error practices

Reply:
Addressed curly braces, but leaving this comment unresolved.

Reply:
Meh. We're good.